### PR TITLE
Fixed Typo and added steps to port examples in EXAMPLE.md

### PR DESCRIPTION
**Pull request by krutikabapat**
Fixed typo in the section "Adding new examples"

### CODE REVIEW

File: doc/readme/EXAMPLES.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example: simple create another `*.sg` file.
+It is extremely simple to add a new example. Just create another `*.sg` file.
```
Comment (gf712) [inline doc/readme/EXAMPLES.md]: I think this was meant to be: "It is extremely simple to add a new example: simply create another `*.sg` file."

File: doc/readme/EXAMPLES.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
...
 We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
```
Comment (gf712) [inline doc/readme/EXAMPLES.md]: Maybe you can write here the steps required to porting the examples? It is described in the issue, but would be easier to have it all here imo

Comment (krutikabapat) [reply doc/readme/EXAMPLES.md]: @gf712 Got it! I will add some steps as mentioned in the issue and will create a new PR. Thank you for your suggestions!

**Comment by krutikabapat** [comment]
@gf712 Can you please review the following steps to be added in EXAMPLES.md. Thanks in advance :D.  
It is extremely simple to add a new example. Just create another `*.sg` file. The following are the steps to port the examples:

1. Add file in ```examples/meta/src/*/*.sg``` (```.sg``` file contains example code in meta-language specific to shogun)  

2. The next step is to build these examples. During build these are parsed and then translated with the (Python) machinery in ```examples/meta/generator/*.py.``` The output is a code listing for each target language defined in ```examples/meta/src/generator/targets/*.json.```   

For python, the command is:   
``` 
make meta_examples
```    
**Note:** This is only available when the cmake option ```BUILD_META_EXAMPLES=ON``` is set, and the Python requirements in ```examples/meta/requirements.txt``` are met.  

For C++, the command is:  
```
make build_cpp_meta_examples
``` 
and then run them as from their folder straight-away (you might have to set environmental variables, see INTERFACES.md)  
```
cd examples/meta/cpp/multiclass
./cpp-multiclass-k_nearest_neighbours
```

**Comment by vigsterkr** [comment]
@krutikabapat plz update the pr instead of discussing things in comments... way more easier to do it/review that is/.

**Comment by krutikabapat** [comment]
@vigsterkr I will update the PR. Please let me know if any other changes are required then :)

File: doc/readme/EXAMPLES.md [hunk 3]
```diff
@@ -35,8 +35,14 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example. Just create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
+It is extremely simple to add a new example: simply create another `*.sg` file.
+We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
```
Comment (vigsterkr) [inline doc/readme/EXAMPLES.md]: `to the new parameter framework`

File: doc/readme/EXAMPLES.md [hunk 4]
```diff
@@ -35,8 +35,14 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
 For details, see `CMakeLists.txt` in `examples/meta/` for details, `generate.py` and `translate.py` in `examples/meta/generator/`.
 
 ## Adding new examples
-It is extremely simple to add a new example. Just create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
+It is extremely simple to add a new example: simply create another `*.sg` file.
+We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).  
+
+The following are the steps required to port all existing examples to meta examples:
```
Comment (vigsterkr) [inline doc/readme/EXAMPLES.md]: `The following steps are required`

File: doc/readme/EXAMPLES.md [hunk 5]
```diff
@@ -35,8 +35,14 @@ Alternatively, you see [INTERFACES.md](INTERFACES.md) on how to run them manuall
...
 
 ## Adding new examples
-It is extremely simple to add a new example. Just create another `*.sg` file.
-We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).
+It is extremely simple to add a new example: simply create another `*.sg` file.
+We are currently porting all existing Python examples in the deprecated folder `examples/undocumented/python_modular` to the new system -- a copy paste [entrance task](https://github.com/shogun-toolbox/shogun/issues/3555).  
+
+The following are the steps required to port all existing examples to meta examples:  
+
+1. Just select any of the existing python examples ```examples/uncodcumented/python_modular```
```
Comment (vigsterkr) [inline doc/readme/EXAMPLES.md:43]: `... existing python examples under`

Comment (vigsterkr) [reply doc/readme/EXAMPLES.md:43]: btw a simple \`this\` is enough it's not a listing block..

**Review by vigsterkr — CHANGES_REQUESTED** [review]

File: doc/readme/EXAMPLES.md (hunk 5 above)
Comment (krutikabapat) [reply doc/readme/EXAMPLES.md:43]: @vigsterkr Really thank you for suggesting changes. I have updated the PR.

Comment (krutikabapat) [reply doc/readme/EXAMPLES.md:43]: @vigsterkr Please let me know if any further updations are required.

**Pull request closed without merging**